CovertInfiltration/Src/CovertInfiltration/Classes/UIStackingIcon.uc

```diff
@@ -0,0 +1,68 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  Xymanek
+//  PURPOSE: This is a reusable panel for showing StackedUIIconData. Mainly used for
+//           faction icons.
+//  NOTE:    The (0,0) position of this panel lies at the center of icon (not at top-left
+//           as other UIPanels do). Please account for that when spawining/positioning
```

Comment:
Is this something we can/should change?

Reply:
The only way I see to fix this is to have the actual icon inside a decorator panel which would make this very confusing and require overriding probably the entire `UIPanel` api....

So I think the answer to the `should` part is definitely `no`

Reply:
Doesn't seem too bad to me

Reply:
Do you mean that it should be simple?